src/frontend/sharedTypes/navigation.ts

```diff
@@ -163,6 +163,11 @@ export type RootStackParamsList = {
   ProjectStatsTurnedOff: undefined;
   EarlyAccess: undefined;
   EarlyAccessOff: undefined;
+  CollaboratorInfo: {
+    deviceId: string;
+    isOwnDevice: boolean;
+    deviceType: DeviceType;
```

Comment:
deviceType does not get returned from the api `useSingleMember`, so the simplest way was to just pass it from the parent.